README.md

```diff
@@ -21,6 +21,10 @@
 
 *Owned by tfukaza, Feb 4 2022*
 
+## yellow
+![yellow](https://singlecolorimage.com/get/A020F0/400x100)
```

Comment:
I think you could check this out: [Comprehensive Markdown Style Guide](https://seesawseeneyewear.com/)

Reply:
no u